homeassistant/components/media_player/songpal.py

```diff
@@ -154,7 +154,7 @@ def available(self):
                 _LOGGER.warning("Got %s volume controls, using the first one",
                                 volumes)
 
-            volume = volumes.pop()
+            volume = volumes.pop(0)
```

Comment:
Can you do `volume = volumes[0]` (lots faster since list is not optimized for popping anything but the last index) or do you need to mutate the list?

Reply:
No need for mutation, so changed it, thanks!